Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -1,7 +1,14 @@
-﻿using System;
+using System;
 using System.Linq;
+#if NOTNET45
```

Comment:
No need to define a new constant, just use `!NET45`.

Reply:
Fixed